exercises/exercise 1/README.md

```diff
@@ -87,3 +89,11 @@ Status: Downloaded newer image for dapworkshop/workshop:latest
 Now the workshop can really start!
 
 **Don't forget to save on your laptop all your exercises before leaving the workshop**.
+Copy/Paste the `SSH` address and use it to `SCP` all your files:
```

Comment:
You should clarify what SCP is for non networking / CLI people. i.e. `SCP` (Secure File Copy)

Reply:
Oh sorry missread your command.